voltixApp/VoltixApp/Views/Vault/EditVaultView.swift

```diff
@@ -76,6 +77,10 @@ struct EditVaultView: View {
         }
     }
     
+    var deviceName: some View {
+        Text(Utils.getLocalDeviceIdentity()).padding(.top, 30)
```

Comment:
Here as well , you should not call `Utils.getLocalDeviceIdentity()` after the vault is already created , use `vault.localPartyID` instead

Reply:
Changing now